Makefile

```diff
@@ -59,19 +57,19 @@ test: generate verify ## Run tests
 
 .PHONY: docker-build
 docker-build: generate ## Build the production docker image
-	bazel run //cmd/manager:manager-image $(BAZEL_DOCKER_ARGS)
+	bazel run //cmd/manager:manager-image $(BAZEL_ARGS)
 
 .PHONY: docker-build-dev
 docker-build-dev: generate ## Build the development docker image
-	bazel run //cmd/manager:manager-image-dev $(BAZEL_DOCKER_ARGS)
+	bazel run //cmd/manager:manager-image $(BAZEL_ARGS) --define=REGISTRY=gcr.io/$(shell gcloud config get-value project)
```

Comment:
We should probably also expose this out as an env var for make, otherwise overriding it requires new contributors to know how to format BAZEL_ARGS.

Reply:
+1 to using a variable instead. I think there are some folks using a different registry.